frontend/src/metabase/dashboard/hoc/DashboardControls.jsx

```diff
@@ -174,6 +174,7 @@ export default ComposedComponent =>
             { preserveParameters: true },
           );
           this.props.fetchDashboardCardData({
+            skipOngoingDashcardRequests: true,
```

Comment:
This is under `_tickRefreshClock` function. So this new parameter `skipOngoingDashcardRequests` is a way to tell the action that we want to skip fetching new dashcards when we know those dashcards are still being loaded.

The previous name I came up with was `skipLoadingDashcards` as in **to skip the dashcards that are being loaded**, but that's ambiguous since it could be interpreted as **to skip loading any dashcards**

Reply:
Fair, this is only for this particular use case. If in the future, we decide to do the same for other scenarios we can rename it to something else then.